frontend/src/hooks/useLookupCodes.tsx

```diff
@@ -46,8 +55,34 @@ function useCodeLookups() {
   };
 
   const getOptionsByType = (type: string) => getByType(type).map(mapLookupCode);
+
+  /**
+   * Return an array of SelectOptions containing property classifications.
+   * @function
+   * @param filter - A filter to determine which classifications will be returned.
+   * @returns
+   */
+  const getPropertyClassificationOptions = (
+    filter?: (value: SelectOption, index: number, array: SelectOption[]) => unknown,
+  ) => {
+    const classifications = getByType(API.PROPERTY_CLASSIFICATION_CODE_SET_NAME);
+    return filter
+      ? (classifications ?? []).map(c => mapLookupCode(c)).filter(filter)
+      : !keycloak.hasClaim(Claims.ADMIN_PROPERTIES)
```

Comment:
slight preference to structuring these as if true then false. optional.

Reply:
I think it does follow that.  `filter == true` use the filter, otherwise do the claim.